mod-audit-server/src/main/java/org/folio/dao/OrderEventDao.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.dao;
+
+import io.vertx.core.Future;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import org.folio.rest.jaxrs.model.OrderAuditEvent;
+
+import java.util.Date;
+
+public interface OrderEventDao {
```

Comment:
Please add java docs
Also need to rename to OrderEventsDao because our handler name is OrderEventsHandler

Reply:
fixed